### PR TITLE
feat: new design

### PR DESCRIPTION
**Pull request by efbarong**
Close task https://modyohq.atlassian.net/browse/FRAMEWORK-349

image

image

image

### CODE REVIEW

**Review by johannrp27 — DISMISSED** [review]

File: src/components/AccountSelectorSlider.tsx [hunk 1]
```diff
@@ -64,8 +64,10 @@ export default function AccountSelectorSlider() {
           rewind: true,
           mediaQuery: 'max',
           breakpoints: {
-            576: {
+            991: {
```
Comment (abe-modyo) [inline src/components/AccountSelectorSlider.tsx:67]: 992.

Comment (abe-modyo) [reply src/components/AccountSelectorSlider.tsx:67]: image

File: src/styles/base.scss [hunk 2]
```diff
@@ -74,50 +71,34 @@
     }
   }
 
-  .account-slider {
-    padding-block: var(--#{$prefix}ref-spacer-4);
-  }
+  // .account-slider {
+  //   padding-block: var(--#{$prefix}ref-spacer-4);
+  // }
```
Comment (abe-modyo) [inline src/styles/base.scss:76]: why? borrar o descomentar ? 🤔

**Review by abe-modyo — CHANGES_REQUESTED** [review]
holi

**Review by abe-modyo — APPROVED** [review]

**Pull request merged**